config/src/config/quorum_store_config.rs

```diff
@@ -39,9 +39,9 @@ impl Default for QuorumStoreConfig {
             channel_size: 1000,
             proof_timeout_ms: 10000,
             batch_request_num_peers: 2,
-            mempool_pulling_interval: 100,
+            mempool_pulling_interval: 200,
```

Comment:
Did we find that 100ms is too often?

Reply:
From the BATCH_CREATION_DURATION, we observe that we create batch every 400ms to 2s depending on how loaded the system is. And for 100 nodes, each node has to create a batch only about twice every second and it would be backpressured rest of the time. So, scheduling the pull every 100ms is just too frequent without doing any useful working.

Reply:
Interested to see how this works with single dc small forge tests. Not that we have to optimize for them in the defaults. If needed we could probably override this for those tests.